Tests/SwiftDocCTests/Test Bundles/MixedLanguageFramework.docc/symbol-graphs/clang/MixedLanguageFramework.symbols.json

```diff
@@ -81,7 +81,16 @@
       "source": "c:@M@TestFramework@objc(cs)MixedLanguageClassConformingToProtocol(im)init",
       "target": "c:@M@TestFramework@objc(cs)MixedLanguageClassConformingToProtocol",
       "targetFallback": null
-    }
+    },
+    {
+      "kind": "memberOf",
+      "source": "c:@CM@TestFramework@objc(cs)MixedLanguageClassConformingToProtocol(im)mixedLanguageMethod",
+      "target": "c:@M@TestFramework@objc(cs)MixedLanguageClassConformingToProtocol",
+      "sourceOrigin": {
+          "identifier": "c:@M@TestFramework@objc(pl)MixedLanguageProtocol(im)mixedLanguageMethod",
+          "displayName": "MixedLanguageProtocol.mixedLanguageMethod()"
+      }
+  },
```

Comment:
We try to avoid making any changes that are for a specific test in any test bundle that's used by other tests. Instead, when a specific test needs a minor modification of a shared test bundle we usually make that test copy the bundle and apply its own modifications there.

For test that mainly change the documentation of an existing test bundle—for example by adding or modifying a documentation extension file—we commonly use something like

```swift
let (_, bundle, context) = try testBundleAndContext(copying: "MixedLanguageFramework") { url in
    // Apply changes here before the context loads the catalog.
    //
    // `url` refers to the copied version of the catalog, 
    // so something like `url.appendingPathComponent("MyNewFile.md")` 
    // refers to a new file inside the catalog
}
```

Some tests that make slight modifications to symbol graph files also use the same patter to modify a symbol graph file by decoding it, modifying it, and encoding it again. See for example `ConstraintsRenderSectionTests.testSingleConstraint()`.

In other cases, if a symbol graph file that reproduces the issue is simple to describe, some tests define a symbol graph in code using `XCTestCase.makeSymbolGraph(moduleName:platform:symbols:relationships:)`.

Reply:
Also, I'm assuming that this is a manually added relationship since Ethan generated the original file 3 years ago (according to the file history) and I don't know if anyone has the source project that this stems from.

If that's true; we should try to avoid making any manual modifications to any compiler emitted symbol graph file because it makes the test data less realistic.

Reply:
Ok I've rewritten the tests to avoid manual changes to the fixture symbol graph files.